### ISSUE TITLE
Hide Primary Media Option on Stories Site

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#4240]
# Story

As a stories site content editor, I would like the ability to include a primary image that will be used for social media and teaser views but have the option to hide it from displaying on the full node so that more vertical oriented photos can be used in the body of the story.

# Additional Info

There may be some display issues if the primary media is not being displayed that will need accounted for.

# Nice to Have

When using a video as the primary media item they would like to have an image that can be used for teasers and social media. This is a nice to have and they can live without this if it can't be completed in a single sprint as part of the primary task.

https://stories.uiowa.edu/iowa-dives-future-water-research

# Proposed solution

- [x] Require the image field (for teaser/social)
- [x] Add boolean to allow editor to not display image on full node
- [x] Create a separate optional media video field 
- [x] Update logic full node to account for if primary media (image/video) exists and boolean is FALSE. Favor video over image if provided.
- [ ] And consider manual or automated migration of videos on existing content (not many).
- [x] Set boolean as FALSE via update for content on all content or account for null/not set in later logic. Set TRUE for any existing content that doesn't have an image.
- [x] Style full node to handle boolean TRUE.

## Above and beyond

- [ ] Provide site with some generic image assets from brand.uiowa.edu or photoshelter to use for stories that don't have enough imagery.

**Comment by makurj** [comment]
I'd like to pair on this.

**Comment by makurj** [comment]
Waiting on user's testing and feedback to figure out if they would like to manually migrate the videos over to the new field in stories and provide thumbnails for them for the teaser view.

**Comment by makurj** [comment]
Completed. Thanks, @joewhitsitt!
